semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/AndroidIncubatingAttributes.java

```diff
@@ -29,20 +29,11 @@ public final class AndroidIncubatingAttributes {
    * <p>The Android lifecycle states are defined in <a
    * href="https://developer.android.com/guide/components/activities/activity-lifecycle#lc">Activity
    * lifecycle callbacks</a>, and from which the {@code OS identifiers} are derived.
-   *
-   * <p>
-   *
-   * @deprecated Replaced by {@code device.app.lifecycle}.
    */
-  @Deprecated public static final AttributeKey<String> ANDROID_STATE = stringKey("android.state");
```

Comment:
I pushed a commit which shows the change to generated source code from this upgrade. 

Removal of `@Deprecated` seems incorrect.

Reply:
yeah, see https://cloud-native.slack.com/archives/C0697EXNTL3/p1739199309470329